src/app/components/SummaryHighlight/SummaryHighlight.examples.tsx

```diff
@@ -0,0 +1,6 @@
+import { Example } from '../examples';
+import SummaryHighlightItem from './SummaryHighlight';
+
+export const LolSummaryHighlightItem: Example = () => (
+  <SummaryHighlightItem highlights={['test, test']} timestamp="1627390756" />
```

Comment:
I would expect that `timestamp` is a number. This number looks very high if we want to specify the seconds from match start.
Let's try to find real-world examples instead of `test`. Maybe `['kill', 'death']`?

Reply:
in my files, the timestamp is a number, weird